include/spirv/unified1/spirv.core.grammar.json

```diff
@@ -4493,6 +4493,77 @@
       "extensions" : [ "SPV_KHR_integer_dot_product" ],
       "version" : "1.6"
     },
+    {
+      "opname" : "OpTypeCooperativeMatrixKHR",
+      "class"  : "Type-Declaration",
+      "opcode" : 4456,
+      "operands" : [
+        { "kind" : "IdResult" },
+        { "kind" : "IdRef",        "name" : "'Component Type'" },
+        { "kind" : "IdScope",      "name" : "'Execution'" },
```

Comment:
Nit: https://github.com/KhronosGroup/SPIRV-Registry/pull/207 calls this "Scope".

Reply:
Done